orchestra/project_api/views.py

```diff
@@ -74,6 +73,11 @@ def project_details_url(request):
 
 @api_endpoint(['GET'])
 def workflow_types(request):
-    workflows = WorkflowVersion.objects.all()
-    workflow_choices = {w.slug: w.name for w in workflows}
-    return {'workflows': workflow_choices}
+    workflows = {
+        w.slug: {
+            'name': w.name,
+            'versions': {v.slug: v.name for v in w.versions.all()}
+        }
+        for w in Workflow.objects.all()
+    }
```

Comment:
I think this should probably just spit out the `workflow.json` and `version.json` manifest for all installed workflows/versions, but this is a simpler change for now. Thoughts? (I'd also consider adding more fields into this gradually, `description` and `certifications` are obvious candidates)